go_tests/bal_test_test/bal_test_group/.gitignore

```diff
@@ -0,0 +1,3 @@
+target
+generated
+Config.toml
```

Comment:
.gitignore and .devcontainer.json files can be removed as they aren't necessary for running the tests